Tentacle/Client.swift

```diff
@@ -312,6 +317,11 @@ public final class Client {
         return fetchMany(.publicRepositories, page: page, pageSize: perPage)
     }
 
+    /// Fetch a content in a specific repository
+    public func content(in repository: Repository, at path: String) -> SignalProducer<(Response, Content), Error> {
```

Comment:
It might make more sense to put the path first. I also think it could just be called `path`.

```
content(path: String, in repository: Repository)
```